core/src/main/java/org/apache/iceberg/MergingSnapshotProducer.java

```diff
@@ -279,7 +284,6 @@ protected void add(ManifestFile manifest) {
           COMMA.join(Iterables.transform(Iterables.filter(deletePaths,
               path -> !deletedFiles.contains(path)),
               CharSequenceWrapper::get)));
-
```

Comment:
Please revert non-functional changes like this. These tend to cause needless git commit conflicts.

Reply:
done.